src/api/.env.example

```diff
@@ -0,0 +1 @@
+DATABASE_URL=
```

Comment:
I would prefer for this example to be in the README.

Reply:
I've done the needful. Please check the README file whenever you get a chance.